corehq/apps/app_manager/detail_screen.py

```diff
@@ -393,7 +389,22 @@ class LateFlag(HideShortHeaderColumn):
 
 @register_format_type('invisible')
 class Invisible(HideShortColumn):
-    pass
+    @property
+    def header(self):
```

Comment:
Could be good to add a comment here to explain why we're giving an 'invisible' column a header value. Something like: 

The header value for 'invisible' columns is used by the sort menu.